README.md

```diff
@@ -12,8 +12,9 @@ Currently, the functionality of **connectivity_check** is also bundled in the ag
 
 ### Flags
 
-* *--host*:  The inventory host as dns name or ip address. Default is "api.openshift.com".
-* *--port*: The inventory port number. Default is 80.
+* *--host*:  **Deprecated, use *--url* instead.** The inventory host as dns name or ip address. Default is "api.openshift.com".
+* *--port*: **Deprecated, use *--url* instead.** The inventory port number. Default is 80.
+* *--url*: The inventory URL, including a schema and optionally a port. Overrides the host and port arguments. If not provided, "http://\<host\>:\<port\>" will be used.
```

Comment:
should it override the cluster value as well?

Reply:
Not related, but I will check.

Reply:
did you add a new issue to follow up? imho we should just provide a url that is:
https://service/api/..clusterid 

then all related options should be deprecated.

Reply:
I don't think we can do it because the base URL is used for multiple endpoints. Just to make sure we're on the same page, this is the base URL of an inventory service API.